src/MissionManager/SpeedSection.cc

```diff
@@ -93,7 +93,7 @@ void SpeedSection::appendSectionItems(QList<MissionItem*>& items, QObject* missi
                                             MAV_CMD_DO_CHANGE_SPEED,
                                             MAV_FRAME_MISSION,
                                             _masterController->controllerVehicle()->multiRotor() ? 1 /* groundspeed */ : 0 /* airspeed */,    // Change airspeed or groundspeed
-                                            _flightSpeedFact.rawValue().toDouble(),
```

Comment:
There shouldn't be any changes at all in this file.

Reply:
my bad! corrected!